src/modules/transaction/components/TransactionDetails/sender.js

```diff
@@ -10,7 +9,6 @@ import ValueAndLabel from './valueAndLabel';
 const Sender = ({ t }) => {
   const { activeToken, transaction, network } = React.useContext(TransactionDetailsContext);
   const validatorName = getValidatorName(transaction, activeToken);
-  const senderLabel = getModuleCommandSenderLabel()[transaction.moduleCommand];
```

Comment:
why was this removed?

Reply:
Already for the wallet connect its would only be a constant that the label would be `Sender`. having  to use `getModuleCommandSenderLabel` to get the summary title for sender based on tx module would be on over kill I think.

Reply:
just taking a second look and yhea you are right